internal/declarative/v2/reconciler.go

```diff
@@ -394,7 +394,7 @@ func (r *Reconciler) checkTargetReadiness(
 
 	resourceReadyCheck := r.CustomReadyCheck
 
-	crStateInfo, err := resourceReadyCheck.Run(ctx, clnt, manifest, target)
+	crStateInfo, err := resourceReadyCheck.Run(ctx, clnt, target)
```

Comment:
`crStateInfo` previously means module CR state, with your change, it's the deployment state. please consider changing to a proper name.